terraform/features-api/vars/staging.tf

```diff
@@ -0,0 +1,13 @@
+# region             = "us-west-2"
```

Comment:
you shouldn't have to worry about commenting var files out. They won't be used b/c you should be specifically passing them in `plan/apply` via the `-var-file=./vars/<name>.tf` argument. Maybe take a look at this document and the last command:

https://github.com/NASA-IMPACT/veda-features-api/blob/main/docs/IACHOWTO.md